spec/app/proxy_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 RSpec.describe ProxyPages do
   before do
     allow(AppDocs).to receive(:apps_with_docs)
-      .and_return([double("App with docs", app_name: "", page_title: "", description: "")])
+      .and_return([double("App with docs", app_name: "", page_title: "", description: "", github_repo_name: "")])
```

Comment:
Should consider if I can use `AppDocs.apps_with_docs` and filtering based on presence. Probably better than duplicating business logic, and means I can avoid adding this line.